### ISSUE TITLE
JS exceptions trying to access code of zones close to antimeridian or north pole

### ISSUE THREAD

**Issue by ilyabo** [teralytics/geohex#17]
```
> t.zoneByLocation(-177.77777777777777, 82.77472507444895,5).code
java.util.NoSuchElementException: key not found: Cell(-2,0)
> t.zoneByLocation(-22.22222222222223, 88.55526595063087, 5).code
java.util.NoSuchElementException: key not found: Cell(-1,-1)

```

Just getting the zone works, but trying to access the code prop leads to exceptions. It would be more reasonable if exceptions were already thrown when calling zoneByLocation. This way it would be easier to track and remove unsupported hexagons.

```
> t.zoneByLocation(-22.22222222222223, 88.55526595063087, 5)
{ hb: 
   { Eb: 300,
     Rb: { rd: [Object], xd: [Object] },
     Yb: { Eb: 300 },
     zc: 5,
     Ee: 1.234567901234568,
     Xi: null,
     ma: false },
  zc: 5,
  Ee: 1.234567901234568 }
> t.zoneByLocation(-22.22222222222223, 88.55526595063087, 5).code
java.util.NoSuchElementException: key not found: Cell(-1,-1)

```

**Comment by shishkin** [comment]
Fixed in version 0.1.87

**Comment by ilyabo** [comment]
woo–hoo! 👍
